monkey/monkey_island/cc/ui/src/components/Main.tsx

```diff
@@ -192,8 +192,10 @@ class AppComponent extends AuthComponent {
   getIslandModeTitle(){
     if(this.state.islandMode === 'ransomware'){
       return this.formIslandModeTitle("Ransomware", faFileCode);
-    } else {
+    } else if(this.state.islandMode === 'advanced') {
       return this.formIslandModeTitle("Custom", faLightbulb);
+    } else {
+      return this.formIslandModeTitle("", faTimes);
```

Comment:
What is this? Isn't it better to not show anything in case of an error than just to show "X" with unknown meaning?